subseas_flex_fcst/config-sample.yaml

```diff
@@ -18,16 +18,12 @@ hcst_filePattern: CFSv2_SubXPRCP_CCAFCST_xvPr_Apr_*.txt
 variable: Precipitation Anomaly
 
 #S,L, target dates options
-numberLeads: 4
-dataLead1: wk1
-lead1: 1
-dataLead2: wk2
-lead2: 8
-dataLead3: wk3
-lead3: 15
-dataLead4: wk4
-lead4: 22
-target_period_length: 7
+leadInit: wk1
```

Comment:
probably safe to assume that taking the first lead by default will be ok in all circumstances. In which case can get rid of that config entry.